src/sage/rings/semirings/tropical_polynomial.py

```diff
@@ -11,10 +11,10 @@
     sage: R.<x> = PolynomialRing(T)
     sage: x.parent()
     Univariate Tropical Polynomial Semiring in x over Rational Field
-    sage: (x + R(3)*x) * (x^2 + x)
-    3*x^3 + 3*x^2
-    sage: (x^2 + R(1)*x + R(-1))^2
-    0*x^4 + 1*x^3 + 2*x^2 + 0*x + (-2)
+    sage: (R(3)*x + R(1)) * (x^2 + x)
+    3*x^3 + 3*x^2 + 1*x
+    sage: (x^2 + x + R(0))^2
+    0*x^4 + 0*x^3 + 0*x^2 + 0*x + 0
```

Comment:
Again, why are you changing these tests? If you want to have different tests, just add them instead.

Reply:
You haven't answered why these tests are changed instead of adding additional ones.

Reply:
The test is changed to make it clearer while testing the same functionality.

Reply:
I would just have both. More examples/tests is not a bad thing. :)